tests/Policy_1.py

```diff
@@ -112,7 +116,7 @@ def testPolicySetget(self):
 
     def testSimpleLoad(self):
         # n = mwid.Citizen_census(0)
-        p = Policy.createPolicy("examples/EventTransmitter_policy.paf")
+        p = Policy.createPolicy(os.path.join(proddir, "examples", "EventTransmitter_policy.paf"))
```

Comment:
The examples directory has not been touched in 5 years and my understanding is that this package is being deprecated at some point. I don't know enough about it to easily know which examples are just examples and which are test data and whether the examples are there because the tests may as well be using example files rather than duplicating them.